networkx/algorithms/operators/product.py

```diff
@@ -291,10 +292,16 @@ def strong_product(G, H):
 
     The strong product $P$ of the graphs $G$ and $H$ has a node set that
     is the Cartesian product of the node sets, $V(P)=V(G) \times V(H)$.
-    $P$ has an edge $((u,v), (x,y))$ if and only if
-    $u==v$ and $(x,y)$ is an edge in $H$, or
-    $x==y$ and $(u,v)$ is an edge in $G$, or
-    $(u,v)$ is an edge in $G$ and $(x,y)$ is an edge in $H$.
+    $P$ has an edge $((u,x), (v,y))$ if any of the following conditions
+    are met:
+    - $u==v$ and $(x,y)$ is an edge in $H$, or
```

Comment:
also no need for "or" if you are not using "if and only if" above
```suggestion
    - $u==v$ and $(x,y)$ is an edge in $H$,
```

Reply:
fixed